lib/org/pepapod/services/harvester/position/default-position-service.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter_geo_poc/org/pepapod/services/services-declaration.dart';
+import 'package:geolocator/geolocator.dart';
+
+class DefaultPositionService implements PositionService {
+
+  static PositionService _instance;
+
+  DefaultPositionService._();
+
+  factory DefaultPositionService.instance() {
+    if (_instance == null) {
+      _instance = DefaultPositionService._();
+    }
+    return _instance;
+  }
+
+  @override
+  Future<Position> getCurrentPosition() {
+    return Geolocator().getCurrentPosition(desiredAccuracy: LocationAccuracy.high);
```

Comment:
Did you tried `LocationAccuracy.bestForNavigation` ?

Reply:
I'll try